docs/source/introduction/ext_cfg_util.rst

```diff
@@ -0,0 +1,73 @@
+The purpose of the VISA external configuration utility
+==============================================================================
```

Comment:
Please do not extend the underline beyond the title

Reply:
OK, fixed